### PR TITLE
docs: Add information that SLURP is not supported by COU

### PR DESCRIPTION
**Pull request by jneo8**

### CODE REVIEW

File: docs/explanation/upgrade.rst [hunk 1]
```diff
@@ -27,6 +27,12 @@ in a sequential manner, performing upgrades on units within a single zone in par
 *all-in-one* method for upgrades. because its charm is designed to maintain service
 availability throughout the upgrade process.
 
+
+SRULP support
```
Comment (samuelallan72) [inline docs/explanation/upgrade.rst:31]:
```suggestion
SLURP support
```

The acronym is for "Skip Level Upgrade Release Process", ref. https://docs.openstack.org/project-team-guide/release-cadence-adjustment.html

File: docs/explanation/upgrade.rst [hunk 2]
```diff
@@ -27,6 +27,12 @@ in a sequential manner, performing upgrades on units within a single zone in par
...
+~~~~~~~~~~~~~
+
+COU follows Charmed OpenStack's upgrade strategy. So SRULP is not supported by COU.
```
Comment (samuelallan72) [inline docs/explanation/upgrade.rst:34]: Acronym needs fixing here too. Also the two sentences may need to be tied together more - it's not clear that charmed openstack does not support SLURP here.

**Review by samuelallan72 — CHANGES_REQUESTED** [review]

File: docs/explanation/upgrade.rst (hunk 1 above)
Comment (jneo8) [reply docs/explanation/upgrade.rst]: Thanks for verify

File: docs/explanation/upgrade.rst [hunk 3]
```diff
@@ -27,6 +27,12 @@ in a sequential manner, performing upgrades on units within a single zone in par
...
+SLURP support
+~~~~~~~~~~~~~
+
+COU aligns with Charmed OpenStack's upgrade strategy. Since Charmed OpenStack does not support SLURP, COU also does not provide support for it.
+
 .. LINKS
 .. _all-in-one: https://docs.openstack.org/charm-guide/latest/admin/upgrades/openstack.html#perform-the-upgrade
-.. _paused-single-unit: https://docs.openstack.org/charm-guide/latest/admin/upgrades/openstack.html#perform-the-upgrade
\ No newline at end of file
+.. _paused-single-unit: https://docs.openstack.org/charm-guide/latest/admin/upgrades/openstack.html#perform-the-upgrade
```
Comment (samuelallan72) [inline docs/explanation/upgrade.rst:38]: What changed on this line?

**Review by samuelallan72 — APPROVED** [review]
Thanks, LGTM

File: docs/explanation/upgrade.rst (hunk 3 above)
Comment (jneo8) [reply docs/explanation/upgrade.rst:38]: No new line at the end.

Comment (samuelallan72) [reply docs/explanation/upgrade.rst:38]: Hmm usually github ui would show that case. Also, if I show the commit from my local git, it doesn't show a change on this line. Weird. :thinking:

**Review by samuelallan72 — APPROVED** [review]

**Review by chanchiwai-ray — APPROVED** [review]

**Pull request merged**